config/backend-config/backend-config.go

```diff
@@ -145,7 +145,12 @@ func (bc *backendConfigImpl) configUpdate(ctx context.Context, statConfigBackend
 
 	bc.curSourceJSONLock.Lock()
 	if !reflect.DeepEqual(bc.curSourceJSON, sourceJSON) {
-		pkgLogger.Infof("Workspace Config changed: %s", workspaces)
+		if len(workspaces) > 0 {
+			pkgLogger.Infof("Workspace Config changed: %d", len(workspaces))
+		} else {
+			pkgLogger.Infof("Workspace Config changed")
+		}
```

Comment:
```suggestion
		pkgLogger.Infof("Workspace Config changed: %d", len(workspaces))
```

It's OK if it prints "Workspace Config changed: 0", right?

Reply:
Yeah, but I found it confusing to get this long in single-tenant setups.

Reply:
Why long? We're just printing the number of workspaces, it's not like it used to be anymore (list of workspaces with `%s`). In any case up to you, it's a super minor thing :+1:

Reply:
sorry: s/long/log

Reply:
Gotcha :+1: